### ISSUE TITLE
Cypress not working with single spa angular Microfrontend 

### ISSUE THREAD

**Issue by GaurNitai-design** [cypress-io/cypress#25978]
### Current behavior

I have upgraded node version from 16.13.0 to 18.13.0. After upgrading I am using single spa for angular Microfrontend. After upgrading node version suddenly it gives error Microfrontend died in status loading source code main.js. 
If I hit main js on system browser it works fine but on Cypress does not works 

### Desired behavior

It should work in cypress 

### Test code to reproduce

Na

### Cypress Version

9.7.0

### Node version

18.13.0

### Operating System

Windows 10

**Comment by mike-plummer** [comment]
Hi @GaurNitai-design , sorry to hear you're having issues. Right now there doesn't seem to be enough information to reproduce the problem on our end.

Please provide a reproducible example of the issue you're encountering. Here are some tips for providing a [Short, Self Contained, Correct, Example](http://sscce.org/) and our own [Troubleshooting Cypress](https://on.cypress.io/troubleshooting) guide. If you aren't able to provide a reproduction case that we can run to observe your issue then we would need logs from the Cypress process and the browser console to help us diagnose what's going on.

**Comment by mike-plummer** [comment]
Unfortunately we have to close this issue due to inactivity. Please comment if there is new information to provide concerning the original issue and we can reopen.

**Comment by AnthonyAngatia** [comment]
Leaving this link here incase anyone is struggling with the same issue as highlighted by @GaurNitai-design .

https://single-spa.js.org/docs/testing/e2e/#configuring-e2e-tests-to-work-with-overrides

**TLDR**: Single-spa have indicated that you need to expose you're micro-frontend to a publicly accessible domain to be able to run e2e tests.
